packages/patternfly-4/react-docs/src/styles/workspace.css

```diff
@@ -0,0 +1,7 @@
+#___gatsby {
```

Comment:
This works, but could also be solved via `wrapper: componentProps => <div className="theme-pf-c-content" style={{height: "100%"}} {...componentProps} />` in `mdxFullscreenTemplate.js` to be consistent with `mdxTemplate.js`.

Reply:
that would only set 100% height on the wrapper, but the ancestors #__gatsby and a child div still need 100% as well

Reply:
Ah -- you're right!